docs/readthedocs/source/doc/UserGuide/known_issues.md

```diff
@@ -0,0 +1,41 @@
+# BigDL Known Issues
+
+## Spark Dynamic Allocation
+
+By design, BigDL does not support Spark Dynamic Allocation mode, and needs to allocate fixed resources. But if your environment has already configured Spark Dynamic Allocation, or stipulated that Spark Dynamic Allocation must be used, you may encounter the following error:
```

Comment:
allocate fixed resources for deep learning model training

Reply:
fixed.